popSimManu.tex

```diff
@@ -474,13 +474,80 @@ \subsection*{Model QC procedures}
 Snakemake pipeline for 1 pop models
-Snakemake pipeline for 2 pop models
 
- In general, used default
-settings and did not attempt to optimize \dadi or \fastsimcoal runs Used Snakemake to
-manage pipeline for two population analysis Results from two population pipeline
-demonstrate that the different methods tested were generally able to recover the
-simulated parameters
+
+
+\subsection*{Workflow for two population analysis}
+Our goal for the two population inference pipeline was to explore two population inference
+on some of the multi-population demographic models implemented in \stdpopsim.
+The models we examined include the human Gutenkunst model (\cite{gutenkunst2009inferring}) as
+well as the \emph{Drosophila} Li & Stephan model (\cite{li2006inferring}). We implemented three
+commonly-used multi-population inference approaches: \dadi (\cite{gutenkunst2009inferring}),
+\fastsimcoal2 (\cite{excoffier2013robust}), and \smcpp (\cite{terhorst2017robust}). Our aim was not
+to exhaustively test the performance of these methods, but rather to explore their behavior
+in two different inference scenarios and demonstrate the potential for this resource to
+eventually be used for more in-depth benchmarking. As a result, these methods were used
+generally with default settings and we did not attempt to optimize their performance or fit
+parameter-rich demographic models. As with the single population inference pipeline,
+we developed a \texttt{snakemake} (\cite{koster2012snakemake}) workflow to manage
+our inference pipeline and maintain reproducibility.
+
+For both \dadi and \fastsimcoal2, our general approach was to fit a two population
+isolation with migration (IM) model with constant population sizes to the simulated
+data from the human and \emph{Drosophila} models. This IM model contains six parameters:
+the ancestral population size, the size of population 1 after split, the size of
+population 2 after split, the divergence time, and two migration rate parameters. Importantly,
+this meant that for both the human and \emph{Drosophila} inference, the fitted model
+did not match the simulated model (Figure \ref{fig:IM_popn_human}C), which, for humans,
+includes three populations with various population size changes as well as exponential
+growth. In the human case, we therefore performed inference solely on the Africa (YRI)
+and Europe (CEU) populations, meaning that the Asia (CHB) population functioned as a ghost
+population that was ignored by our inference. Our motivation for this simple
+approach was to mimic the typical approach of two population inference on empirical
+data, where the user is not aware of the 'true' underlying demography and the inference
+model is often misspecified. To ground-truth our inference approach, we also conducted
+inference on a generic IM model that was identical to the model used for inference.
+% CK: I have a figure of the generic IM results in the works that we can include in the supplement
```

Comment:
yeah i didn't get in to generic models for the 1 population case. this will be something we should hash out-- do we actually want to present generic results? to what level of details?

Reply:
I think theyre perhaps worth including in the supplement as a sanity check. Not sure how needed that is for the 1 pop stuff but I think its worthwhile to show the case where the model is correctly specified for the 2 pop case.

Reply:
cool